umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -1,67 +1,114 @@
+  and microservices.
 weight: 1000
 edition: community
 aliases:
   - /docs/features/unified-namespace
 ---
 
-The Unified Namespace is an event-driven architecture that allows for seamless communication between nodes in a network. It operates on the principle that all data, regardless of whether there is an immediate consumer, should be published and made available for consumption. This means that any node in the network can work as either a producer or a consumer, depending on the needs of the system at any given time.
+The Unified Namespace is a centralized, standardized, event-driven data
+architecture that enables for seamless integration and communication across
+various devices and systems in an industrial environment. It operates on the
+principle that all data, regardless of whether there is an immediate consumer,
+should be published and made available for consumption. This means that any
+node in the network can work as either a producer or a consumer, depending on
+the needs of the system at any given time.
 
-To use any functionalities of the United Manufacturing Hub, you need to use the Unified Namespace as well. More information can be found in our [Learning Hub on the topic of Unified Namespace](https://learn.umh.app/lesson/introduction-into-it-ot-unified-namespace/).
+This architecture is the foundation of the United Manufacturing Hub, and you
+can read more about it in the [Learning Hub article](https://learn.umh.app/lesson/introduction-into-it-ot-unified-namespace/).
 
 ## When should I use it?
 
-An application consists always out of multiple building blocks. To connect those building blocks, one can either exchange data between them through databases, through service calls (such as [REST](https://learn.umh.app/lesson/introduction-into-it-ot-https-rest/)), or through a message broker.
+The Unified Namespace is the ideal solution for exchanging data between
+applications and devices. It effectively eliminates the complexity of spaghetti
+diagrams and enables real-time data processing.
 
-{{% notice tip %}}
-**Opinion:** We think for most applications in manufacturing, communication via a message broker is the best choice as it prevents spaghetti diagrams and allows for real-time data processing. For more information about this, you can [check out this blog article](https://learn.umh.app/blog/comparing-mqtt-brokers-for-the-industrial-iot/#message-brokers-and-mqtt).
-{{% /notice %}}
-
-In the United Manufacturing Hub, each single piece of information / "message" / "event" is sent through a message broker, which is also called the Unified Namespace.
+While data can be shared through databases,
+[REST APIs](https://learn.umh.app/lesson/introduction-into-it-ot-https-rest/),
+or message brokers, we believe that a message broker approach is most suitable
+for most manufacturing applications. Consequently, every piece of information
+within the United Manufacturing Hub is transmitted via a message broker.
```

Comment:
aybe also: why MQTT AND Kafka (and not a single one): view https://learn.umh.app/blog/tools-techniques-for-scalable-data-processing-in-industrial-iot/